packages/storage-documents/src/on-demand-page-scan-result.ts

```diff
@@ -71,3 +71,7 @@ export interface OnDemandPageScanRunResult {
     pageTitle?: string;
     pageResponseCode?: number;
 }
+
+export interface PartialOnDemandPageScanResult extends Partial<OnDemandPageScanResult> {
+    id: string;
```

Comment:
also add partitionKey

Reply:
these two should be mandatory fields to avoid cross partition query

Reply:
We can get the partitionKey using the id and itemType, so I don't think we need to require it here

Reply:
you need to validate just querying for id doesnt result in cross partition query